src/app/backend/replicationcontrollerlist.go

```diff
@@ -51,7 +51,7 @@ type ReplicationController struct {
 	// Label of this Replication Controller.
 	Labels map[string]string `json:"labels"`
 
-	// Aggregate information about pods belonging to this repolica set.
+	// Aggregate information about pods belonging to this replica set.
```

Comment:
At the moment it's `replication controller`. Unfortunately, not all of the comments were updated during rename.